pkgs/top-level/python-aliases.nix

```diff
@@ -75,6 +75,7 @@ mapAliases ({
   MechanicalSoup = mechanicalsoup; # added 2021-06-01
   pam = python-pam; # added 2020-09-07.
   PasteDeploy = pastedeploy; # added 2021-10-07
+  pep257 = pydocstyle; # added 2022-03-28
```

Comment:
The commit message is not that nice here. How about : `python310Packages.pep257: merged into pydocstyle`?

Reply:
Updated